### ISSUE TITLE
bash scripts/evaluation/run_dataset_cache.sh

### ISSUE THREAD

**Issue by yongli2023** [wjl2244/MeanFuser#14]
(meanfuser)  ✘  🐍 meanfuser  liqw@ubuntu  ~/code/E2E/navsim_workspace/MeanFuser   main ±  bash scripts/evaluation/run_dataset_cache.sh

[2026-07-21 15:34:58,796][__main__][INFO] - Global Seed set to 0
Seed set to 0
[2026-07-21 15:34:58,982][__main__][INFO] - Building Worker
[2026-07-21 15:35:00,585][navsim.planning.utils.multithreading.worker_ray_no_torch][INFO] - Not using GPU in ray
[2026-07-21 15:35:00,585][navsim.planning.utils.multithreading.worker_ray_no_torch][INFO] - Starting ray local!
2026-07-21 15:35:03,984 INFO worker.py:2012 -- Started a local Ray instance.
/home/liqw/env/miniconda3/envs/meanfuser/lib/python3.9/site-packages/ray/_private/worker.py:2051: FutureWarning: Tip: In future versions of Ray, Ray will no longer override accelerator visible devices env var if num_gpus=0 or num_gpus=None (default). To enable this behavior and turn off this error message, set RAY_ACCEL_ENV_VAR_OVERRIDE_ON_ZERO=0
  warnings.warn(
[2026-07-21 15:35:07,955][nuplan.planning.utils.multithreading.worker_pool][INFO] - Worker: RayDistributedNoTorch
[2026-07-21 15:35:07,955][nuplan.planning.utils.multithreading.worker_pool][INFO] - Number of nodes: 1
Number of CPUs per node: 64
Number of GPUs per node: 0
Number of threads across all nodes: 64
[2026-07-21 15:35:07,955][__main__][INFO] - Building SceneLoader
Error executing job with overrides: ['train_test_split=navtest', 'agent=transfuser_agent', 'experiment_name=debug', 'cache_path=/home/liqw/code/E2E/navsim_workspace/cache/traintest_v1_cache', 'output_dir=exp/debug']
Error locating target 'navsim.agents.transfuser.transfuser_agent.TransfuserAgent', see chained exception above.
full_key: agent

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
